src/components/SingleTrikotActual.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react'
+import {List, Image,Container} from "semantic-ui-react";
+import { connect } from 'react-redux'
+import store from "../store";
+import * as riderStageConnectionActions from "../actions/riderStageConnectionsActions";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import of 'riderStageConnectionActions'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1128)